### PR TITLE
Cleaned up and used Carbon

### PR DESCRIPTION
**Pull request by rimi-itk**
* Fixes issue with time format.

### CODE REVIEW

File: Services/AbstractExporter.php [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
 require_once __DIR__ . '/../vendor-plugin/autoload.php';
 
-use Leantime\Domain\Auth\Services\Auth;
+
```
Comment (jekuaitk) [inline Services/AbstractExporter.php]: Why the extra newline?

File: Services/AbstractExporter.php [hunk 2]
```diff
@@ -64,27 +65,19 @@ public function export(array $criteria = [], array $options = []): void
      * Get datetime based on user's date format.
      *
      * @param  string|null $value
```
Comment (jekuaitk) [inline Services/AbstractExporter.php]: Should we add the new parameter `$default` here?

**Review by jekuaitk — CHANGES_REQUESTED** [review]

File: Services/AbstractExporter.php (hunk 1 above)
Comment (rimi-itk) [reply Services/AbstractExporter.php]: It's was a mistake!

File: Services/AbstractExporter.php (hunk 2 above)
Comment (rimi-itk) [reply Services/AbstractExporter.php]: Well, the coding standards don't require it, so I've removed `@param  string|null $value` too. The coding standards require the `@return` annotation.

**Review by jekuaitk — APPROVED** [review]

**Pull request merged**